src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -287,23 +285,23 @@ function getVenvActivateCommand(venvName: string, platform: NodeJS.Platform): st
 }
 
 async function getPythonAlias(): Promise<string> {
-    for (const key of Object.keys(PythonAlias)) {
-        const alias: PythonAlias = <PythonAlias>PythonAlias[key];
-        const errorMessage: string | undefined = await validatePythonAlias(alias);
+    const aliasesToTry: string[] = ['python3.6', 'py -3.6', 'python3', 'python', 'py'];
+    const globalPythonPathSetting: string | undefined = getGlobalFuncExtensionSetting('pythonPath', 'python');
+    if (globalPythonPathSetting) {
+        aliasesToTry.unshift(globalPythonPathSetting);
+    }
+
+    for (const alias of aliasesToTry) {
+        // Validate max version when silently picking the alias for the user
+        const errorMessage: string | undefined = await validatePythonAlias(alias, true /* validateMaxVersion */);
         if (!errorMessage) {
             return alias;
         }
     }
 
-    const enterPython: MessageItem = { title: localize('enterPython', 'Enter Python Path') };
-    const pythonMsg: string = localize('pythonVersionRequired', 'Python {0} is required to create a Python Function project and was not found.', minPythonVersionLabel);
-    const result: MessageItem | undefined = await window.showErrorMessage(pythonMsg, { modal: true }, enterPython);
-    if (!result) {
-        throw new UserCancelledError();
-    } else {
-        const placeHolder: string = localize('pyAliasPlaceholder', 'Enter the Python alias (if its in your PATH) or the full path to your Python executable.');
-        return await ext.ui.showInputBox({ placeHolder, validateInput: validatePythonAlias });
-    }
+    const prompt: string = localize('pyAliasPlaceholder', 'Enter the alias or full path of the Python "{0}" executable to use.', minPythonVersionLabel);
+    // Don't validate max version when prompting (because the Functions team will assumably support 3.7+ at some point and we don't want to block people from using that)
+    return await ext.ui.showInputBox({ prompt, validateInput: validatePythonAlias });
```

Comment:
Should we validate their alias before returning it?

Reply:
It already has `validateInput: validatePythonAlias`